### PR TITLE
Restore some dependences and methods of TestWebSocketClient to make it possible be used by agent

### PR DESCRIPTION
**Pull request by rubenvp8510**

### CODE REVIEW

File: hawkular-command-gateway-itest/pom.xml [hunk 1]
```diff
@@ -32,11 +32,19 @@
...
     <debug.wildfly.argLine />
     <version.com.squareup.okhttp>2.4.0</version.com.squareup.okhttp>
+    <version.com.samskivert.jmustache>1.11</version.com.samskivert.jmustache>
   </properties>
 
   <dependencies>
 
     <dependency>
+      <groupId>com.samskivert</groupId>
+      <artifactId>jmustache</artifactId>
```
Comment (pilhuhn) [inline hawkular-command-gateway-itest/pom.xml]: This is problematic to me, as it has no standard OSI approved license.
Also I wonder if it is really needed as it is only used in one place.

Comment (rubenvp8510) [inline hawkular-command-gateway-itest/pom.xml]: @pilhuhn It is used by several tests on agent to resolve a `{{sessionId}}` placeholder when the method `expectText` is used to test a response.

In order to remove this dependency we need to modify those tests.

File: hawkular-command-gateway-itest/pom.xml [hunk 2]
```diff
@@ -32,6 +32,7 @@
     <debug.failsafe.argLine />
     <debug.wildfly.argLine />
     <version.com.squareup.okhttp>2.4.0</version.com.squareup.okhttp>
+    <version.com.samskivert.jmustache>1.11</version.com.samskivert.jmustache>
```
Comment (pilhuhn) [inline hawkular-command-gateway-itest/pom.xml]: Is that still needed?

Comment (rubenvp8510) [inline hawkular-command-gateway-itest/pom.xml]: Not really, I'll remove it.

**Comment by pilhuhn** [comment]
Thank you

**Pull request merged**